src/drivers/uavcan/dsdl/com/hex/equipment/indication/20001.ButtonCommand.uavcan

```diff
@@ -0,0 +1,6 @@
+#
+# Nodes that are capable of producing sounds should obey.
```

Comment:
This is in a manufacturer namespace (Hex). It sounds like this should be in a standard namespace instead and be vendor-neutral.

Reply:
This part can be deleted